src/main/java/edu/whimc/portals/Main.java

```diff
@@ -21,14 +22,18 @@
 public class Main extends JavaPlugin {
 
     public static final String PERM_PREFIX = "whimc-portals";
+    public static final Material TOOL_MATERIAL = Material.WOODEN_SWORD;
 
-    private MyConfigManager manager;
     private static MyConfig portalData;
     private LocationSaver locationSaver;
 
+    public static Main getInstance() {
```

Comment:
I'm not the biggest fan of statically accessing the main class. I know this is a bit contradictory given my comment about making Portal/Destination managers, but that's just the personal style I use.

Reply:
The reason why I like it is for those managing classes can be accessed with that single static method, so there are practically no other public static fields and everything is retrieved from the same place, and for logging because I don't want to be five references removed from the Main class and have to pass the plugin through everything just to access the logger (or managers within Main). I guess it just depends on how many static fields you want, I'm generally for few as possible.